packages/compiler/test/config/config.ts

```diff
@@ -65,47 +51,48 @@ describe("compiler: config file loading", () => {
       await assertLoadFromFolder("yaml-json");
     });
 
-    it("deep clones defaults when not found", async () => {
-      let config = await loadTestConfig("empty");
-      config.plugins.push("x");
-      config.emitters["x"] = true;
-      config.lint.extends.push("x");
-      config.lint.rules["x"] = "off";
-
-      config = await loadTestConfig("empty");
-      deepStrictEqual(config, {
-        plugins: [],
-        diagnostics: [],
-        emitters: {},
-        lint: {
-          extends: [],
-          rules: {},
-        },
-      });
-    });
-
-    it("deep clones defaults when found", async () => {
-      let config = await loadTestConfig("yaml");
-      config.plugins.push("x");
-      config.emitters["x"] = true;
-      config.lint.extends.push("x");
-      config.lint.rules["x"] = "off";
-
-      config = await loadTestConfig("yaml");
-      deepStrictEqual(config, {
-        plugins: ["foo"],
-        diagnostics: [],
-        emitters: {
-          "foo:openapi": true,
-        },
-        lint: {
-          extends: [],
-          rules: {
-            "some-rule": "on",
-          },
-        },
-      });
-    });
+    // TODO: reenable when there is nested config
+    // it("deep clones defaults when not found", async () => {
+    //   let config = await loadTestConfig("empty");
+    //   config.plugins.push("x");
+    //   config.emitters["x"] = true;
```

Comment:
Why can't this still be tested by mutating emitters and removing the parts that don't exist anymore?

Reply:
we can indeeed just keep a simpler version of this test. Will update